docs/ethpm.rst

```diff
@@ -259,7 +259,7 @@ Examples...
 
 - ``ethpm://packages.zeppelinos.eth/owned@1.0.0``
 
-- ``ethpm://0x582AC4D8929f58c217d4a52aDD361AE470a8a4cD:1/ethregistrar@1.0.0``
+- ``ethpm://0xFe89cc7aBB2C4183683ab71653C4cdc9B02D44b7:1/ethregistrar@1.0.0``
```

Comment:
I'd leave out this whole `ethpm` file, mostly because it doesn't have anything to do with ENS so it doesn't make sense to change this to the ENS address. We can change these to the spoofed address from the previous PR.

Reply:
Sounds good, I'll revert these in this file to what it was prior to the change in #2594 .

Reply:
oops I meant #3014... let's use the spoofed addresses from that PR

Reply:
✅